tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -1,56 +0,0 @@
-/*
- * This file is part of the KubeVirt project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * Copyright 2023 Red Hat, Inc.
- *
- */
-
-package main
-
-import (
-	"github.com/kubevirt/cluster-network-addons-operator/pkg/monitoring"
-	metricsparser "github.com/kubevirt/cluster-network-addons-operator/tools/metrics-parser"
-	"github.com/kubevirt/monitoring/pkg/metrics/parser"
-	dto "github.com/prometheus/client_model/go"
-)
-
-// This should be used only for very rare cases where the naming conventions that are explained in the best practices:
-// https://sdk.operatorframework.io/docs/best-practices/observability-best-practices/#metrics-guidelines
-// should be ignored.
-var excludedMetrics = map[string]struct{}{}
```

Comment:
When working on the linter Shirly asked to leave this var in order to make sure there is no metrics added there. not sure if we still need it @sradco WDTY?

Reply:
Yes. We should keep this.

Reply:
In case there is a metric that for some reason we need to name in a way that is not using the best practices.
It needs to be in the same way across all sub operators.
We have a few metrics like that in other operators.

Reply:
re-added